bundle/DependencyInjection/Compiler/IoStorageWhitelistPass.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\DependencyInjection\Compiler;
+
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\ConfigResolver;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use function array_merge;
+use function array_search;
+
+class IoStorageWhitelistPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        $scopes = array_merge(
+            [ConfigResolver::SCOPE_DEFAULT],
+            $container->getParameter('ezpublish.siteaccess.list')
+        );
+
+        foreach ($scopes as $scope) {
+            if ($container->hasParameter("ezsettings.{$scope}.io.file_storage.file_type_blacklist")) {
+                $blacklist = $container->getParameter("ezsettings.{$scope}.io.file_storage.file_type_blacklist");
+                $index = array_search('svg', $blacklist, true);
+                if ($index !== false) {
+                    unset($blacklist[$index]);
+                }
+                $container->setParameter("ezsettings.{$scope}.io.file_storage.file_type_blacklist", $blacklist);
```

Comment:
This can go inside the `if ($index !== false)` clause.

Reply:
Updated in fbbc63518ed9e1d9eb303395b4cdc031d60ce3f3